overlord/auth/auth.go

```diff
@@ -208,8 +209,30 @@ func LoginCaveatID(m *macaroon.Macaroon) (string, error) {
 func (ma *MacaroonAuthenticator) Authenticate(r *http.Request) {
 	var buf bytes.Buffer
 	fmt.Fprintf(&buf, `Macaroon root="%s"`, ma.Macaroon)
```

Comment:
Since macaroon serialization output is a base64 encoded string, this would be ok, correct?
I can change it if you think `%q` would be better.